app/views/layout.slim

```diff
@@ -2,12 +2,24 @@ doctype html
 html
 	head
 		title Tint
+		link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:100,100i,300,300i,400,400i,700,700i,900,900i"
 		link rel="stylesheet" href="/assets/tint.css"
+		link rel="shortcut icon" href="/favicon.ico" type="image/x-icon"
```

Comment:
I made a favicon.ico file, with the all the different sizes. Is that not the right type?